### ISSUE TITLE
IDE0040 (add accessibility modifiers) false positive for VB module constructor

### ISSUE THREAD

**Issue by mavasani** [dotnet/roslyn#45962]
**Version Used**: Latest 16.8 dogfood build

**Steps to Reproduce**:

```vb
Public Module M
    Sub New()
    End Sub
End Module
```

**Expected Behavior**:
No IDE0040 on constuctor

**Actual Behavior**:
Code fix is offered with IDE0040. Applying the fix leads to compiler error `BC30480` - `Shared 'Sub New' cannot be declared 'Private'.`

![image](https://user-images.githubusercontent.com/10605811/87433433-28e7b580-c59e-11ea-83ce-406fe96620be.png)

**Comment by sharwell** [comment]
Duplicate of #44076
